db_lesson.sql

```diff
@@ -0,0 +1,467 @@
+db_lesson_work2
+
+
+MariaDB [(none)]> show databases;
++--------------------+
+| Database           |
++--------------------+
+| db_lesson          |
+| information_schema |
+| mysql              |
+| performance_schema |
+| phpmyadmin         |
+| test               |
++--------------------+
+6 rows in set (0.001 sec)
```

Comment:
[全体]
`.sql`拡張子のファイルは、SQL文を記載する用のファイルでして、
用途としては、「 SQL文を事前に書いておき、ファイルを読み込んで一気に実行」するのに用いられます。

ここに日本語やSQL文以外の文章が書いてあるとうまく読み込めずにエラーになってしまうので、 SQL文以外は書かない、もしくは`-- `でコメントアウトしておくようにしましょう！

> SQL文ではないもの：
> - コマンド先頭の`MariaDB [(none)]> `（これはMariaDBの入力モードになっていることを示す画面表示です）
> - 実行結果のメモや、その他のメモなど。残したいものがある場合は**コメントアウト**

```suggestion
show databases;

-- +--------------------+
-- | Database           |
-- +--------------------+
-- | db_lesson          |
-- | information_schema |
-- | mysql              |
-- | performance_schema |
-- | phpmyadmin         |
-- | test               |
-- +--------------------+
-- 6 rows in set (0.001 sec)
```

Reply:
.sql拡張子のファイルに「 SQL文を事前に書いておき、ファイルを読み込んで一気に実行」する意図があるとは知りませんでした。読み込む場合はGit bush側で指定するのでしょうか。

Reply:
読み込む場合はGit bush側で指定します👍